test/parser/mixins.spec.js

```diff
@@ -25,6 +25,14 @@ describe('Parser', () => {
             expect(root.first.type).to.eql('atrule');
         });
 
+        it('should parse parametric mixins', () => {
+            const root = parse('.foo(@bar) { color: @bar;}', {mixinsAsAtRules: true, innerMixinsAsRules: true});
+
+            expect(root.first.type).to.eql('atrule');
+            expect(root.first.params[0].type).to.eql('atrule');
```

Comment:
going down that road will mean that postcss-less is a monkeypatch (not in a bad sense at all) more than a true syntax (postcss-scss for example, is a true syntax). if that's your plan moving forward, please consider renaming the module and opening the door for a true syntax in the future.